ansible_ai_connect/ai/api/model_pipelines/wca/pipelines_onprem.py

```diff
@@ -173,6 +174,16 @@ def self_test(self):
         raise NotImplementedError
 
 
+@Register(api_type="wca-onprem")
+class WCAOnPremRoleGenerationPipeline(WCAOnPremPipeline, WCABaseRoleGenerationPipeline):
```

Comment:
Do you need this class?

`WCABaseRoleGenerationPipeline` implements it and the default for "on prem" would be the NOP implementation.

Reply:
Hi Michael, it is a placeholder for (near) future work. Once we got the API from IBM we will implement those Pipelines properly.